src/plugins/github/heuristics/mentionsAuthorReference.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+
+import * as MapUtil from "../../../util/map";
+import * as RV from "../relationalView";
+import * as N from "../nodes";
+
+// MentionsAuthorReferences are inferred edges between posts.  If a post
+// contains an @-mention of a user, and that user has authored any posts
+// in the same thread, then we create MentionsAuthorReferences from the
+// mentioning post to any posts authored by the mentioned author.  A
```

Comment:
Mind being a bit more explicit and writing “any posts authored by the
mentioned author within the same thread”? The fact that this is
thread-scoped is only hinted at in the introductory sentence.